inttest/metrics/metrics_test.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2021 k0s authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package metrics
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/k0sproject/k0s/inttest/common"
+)
+
+type MetricsSuite struct {
+	common.FootlooseSuite
+}
+
+func (s *MetricsSuite) TestK0sGetsUp() {
+	s.NoError(s.InitController(0))
+	s.NoError(s.RunWorkers())
+
+	kc, err := s.KubeClient(s.ControllerNode(0))
+	s.NoError(err)
+
+	err = s.WaitForNodeReady(s.WorkerNode(0), kc)
+	s.NoError(err)
+	/*
+		pods, err := kc.CoreV1().Pods("kube-system").List(context.TODO(), v1.ListOptions{
+			Limit: 100,
+		})
+		s.NoError(err)
+
+		podCount := len(pods.Items)
+
+		s.T().Logf("found %d pods in kube-system", podCount)
+		s.Greater(podCount, 0, "expecting to see few pods in kube-system namespace")
+
+		s.T().Log("waiting to see CNI pods ready")
+		s.NoError(common.WaitForKubeRouterReady(kc), "CNI did not start")
+	*/
+	cfg, err := s.GetKubeConfig(s.ControllerNode(0))
```

Comment:
maybe just remove the commented code

Reply:
oh, i forgot to clean that up. will remove it. thanks!